Actions.java

```diff
@@ -0,0 +1,102 @@
+import java.awt.Dimension;
+import java.awt.Graphics;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
+import java.io.*;
+import java.util.Scanner;
+import javax.swing.*;
+import javax.swing.text.Position;
+import javax.swing.text.StyledDocument;
+public class Actions {

```

Comment:
Way to break up the actions from the window. I wasn't able to find an easy way to do this, so good job!